dockerfiles/centos/is-analytics/Dockerfile

```diff
@@ -0,0 +1,79 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+# set to latest Centos
+FROM centos:7
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org”
+
+# set user configurations
+ARG USER=wso2carbon
+ARG USER_ID=802
+ARG USER_GROUP=wso2
+ARG USER_GROUP_ID=802
+ARG USER_HOME=/home/${USER}
+# set dependant files directory
+ARG FILES=./files
+# set jdk configurations
+ARG JDK=jdk1.8.0*
+ARG JAVA_HOME=${USER_HOME}/java
+# set wso2 product configurations
+ARG WSO2_SERVER=wso2is-analytics
+ARG WSO2_SERVER_VERSION=5.5.0
+ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
+
+RUN yum check-update && \
+    DEBIAN_FRONTEND=noninteractive yum install --downloadonly \
+    nc && \
+    rm -rf /var/cache/yum/* && \
+    >> /etc/bash.bashrc \
+    ; echo "\
+    Welcome to WSO2 Docker resources.\n\
+    The Docker container contains the WSO2 product with its latest updates, which are under the End User License Agreement (EULA) 2.0.\n\
+    \n\
+    Read more about EULA 2.0 (https://wso2.com/licenses/wso2-update/2.0).\n"\
+    > /etc/motd
+
+# create a user group and a user
+RUN groupadd --system -g ${USER_GROUP_ID} ${USER_GROUP} && \
+    useradd --system --create-home --home-dir ${USER_HOME} --no-log-init -g ${USER_GROUP_ID} -u ${USER_ID} ${USER}
+
+# copy the jdk and wso2 product distributions to user's home directory
+COPY --chown=wso2carbon:wso2 ${FILES}/${JDK} ${USER_HOME}/java
+COPY --chown=wso2carbon:wso2 ${FILES}/${WSO2_SERVER_PACK}/ ${USER_HOME}/${WSO2_SERVER_PACK}/
+COPY --chown=wso2carbon:wso2 init.sh ${USER_HOME}/
+COPY --chown=wso2carbon:wso2 ${FILES}/mysql-connector-java-*-bin.jar ${USER_HOME}/${WSO2_SERVER_PACK}/repository/components/lib/
+COPY --chown=wso2carbon:wso2 ${FILES}/dnsjava-*.jar ${USER_HOME}/${WSO2_SERVER_PACK}/repository/components/lib/
+COPY --chown=wso2carbon:wso2 ${FILES}/kubernetes-membership-scheme-*.jar ${USER_HOME}/${WSO2_SERVER_PACK}/repository/components/dropins/
+# set temporary location for artifacts to be persisted
+COPY --chown=wso2carbon:wso2 ${FILES}/${WSO2_SERVER_PACK}/repository/conf/analytics ${USER_HOME}/wso2-tmp/analytics
+
+# set the user and work directory
+USER ${USER_ID}
+WORKDIR ${USER_HOME}
+
+# set environment variables
+ENV JAVA_HOME=${JAVA_HOME} \
+    PATH=$JAVA_HOME/bin:$PATH \
+    WSO2_SERVER_HOME=${WSO2_SERVER_HOME} \
+    WORKING_DIRECTORY=${USER_HOME}
+
+# expose ports
+
```

Comment:
Shall we remove this extra line?

Reply:
@DilanUA this is fixed in https://github.com/wso2/docker-is/pull/73.